book/homework/new-list.md

```diff
@@ -45,6 +45,12 @@ const addLadyDev = function (ladyDev) {
 </details>
    {% endhint %}
 
+<!-- trick markdown to give me a little space between these two sections of text -->
+## 
+{% hint style='danger' %}
+If you didn't use the help section, you'll want to rename the function you created in the previous step to `addLadyDev` and the paramater to `ladyDev` before continuing.
+{% endhint %}

```

Comment:
Didn't know if it will be confusing to attendees doing the homework, if they create their own function & parameter & then we refer to it by the name in the hint section for the rest of the steps.

Or we can leave this bit out & just let them troubleshoot themselves 🤷‍♀